client/extensions/woocommerce/woocommerce-services/views/label-settings/label-settings.js

```diff
@@ -38,6 +41,7 @@ import {
 class ShippingLabels extends Component {
 	componentWillMount() {
 		this.setState( { expanded: this.isExpanded( this.props ) } );
+		this.onVisibilityChange = this.onVisibilityChange.bind( this );
```

Comment:
This line shouldn't be necessary.

Reply:
Looks like you're right, but how does it end up with the right context?

Reply:
IIRC it's part of the Stage 2 ES proposal stuff brought in with our Babel config. Arrow functions are autobound to this.

Reply:
Oh right, it's an arrow function! 👍